easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb

```diff
@@ -36,7 +36,7 @@ configopts += '--with-dft-flavor=libxc '
 configopts += '--with-libxc-incs="-I$EBROOTLIBXC/include" '
 configopts += '--with-libxc-libs="-L$EBROOTLIBXC/lib -lxcf90 -lxc" '
 
-
+#
```

Comment:
please remove this `#`, so the file will be identical to the existing one (which is equivalent to removing it from this PR)

Reply:
done...